app/assets/stylesheets/application/_user_profile.scss

```diff
@@ -0,0 +1,29 @@
+.profile-picture {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.mu-avatar-field {
+  width: 0.1px;
+  height: 0.1px;
+  opacity: 0;
+  overflow: hidden;
+  position: absolute;
+  z-index: -1;
+}
+
```

Comment:
@luchotc @cucusita @fedescarpa I would like to get your comments about the proposed UI.

Reply:
I would prefer that the upload is triggered by clicking in the picture.
Something like this would be great: 
![Screenshot from 2019-04-17 11-35-02](https://user-images.githubusercontent.com/11860076/56296840-a572b680-6105-11e9-90a3-8971445ecb3c.png)

Reply:
@luchotc OK, I'll try to match that design

Reply:
Done! https://youtu.be/pLtiQAbfA28